app/views/miq_request/_prov_field.html.haml

```diff
@@ -87,9 +87,9 @@
             - else
               = options[field]
           - unless field_hash[:notes_display] == :hide || field_hash[:notes].blank?
-            -#  Display notes if available
+            -# Display notes if available
             - if field == :vm_name
-              -#  Removing h() so that '<font color='red'>' can be displayed in field
+              -# Removing h() so that '<font color='red'>' can be displayed in field
               = field_hash[:notes]
             - else
               = field_hash[:notes]
```

Comment:
what is the point if the if/else if it always does `= field_hash[:notes]`?  Can we squash lines 90-95 into a single line of code?